lib/cli/src/commands/app/deploy.rs

```diff
@@ -668,7 +668,12 @@ pub async fn wait_app(
                             if !quiet {
                                 eprintln!();
                             }
-                            eprintln!("{} Deployment complete", "𖥔".yellow().bold());
+                            if !(res.status().is_success() || res.status().is_redirection()) {
+                                eprintln!("The app version was deployed correctly, but fails with a non-success status code of {}", res.status())
+                            } else {
+                                eprintln!("{} Deployment complete", "𖥔".yellow().bold());
```

Comment:
Shouldn't this be green, and the above yellow?

Reply:
I think we decided in 4.3.0 to have `"𖥔".yellow().bold()`.  I'll make line 672 yellow.